front-end/src/app/reports/transactions/transaction-list/transaction-disbursements/transaction-disbursements.component.ts

```diff
@@ -9,13 +9,17 @@ import { TransactionListTableBaseComponent } from '../transaction-list-table-bas
 import { Store } from '@ngrx/store';
 import { ReportService } from 'app/shared/services/report.service';
 import { DateUtils } from 'app/shared/utils/date.utils';
+import { Report, ReportTypes } from 'app/shared/models/report.model';
 
 @Component({
   selector: 'app-transaction-disbursements',
   templateUrl: './transaction-disbursements.component.html',
   styleUrls: ['../../transaction.scss'],
 })
 export class TransactionDisbursementsComponent extends TransactionListTableBaseComponent implements OnInit {
+  @Input() report?: Report;
```

Comment:
Solid choice making this this start as `undefined`!  Now if we use this outside the context of a report, it still works as expected